app/importers/planning_applications_importer.rb

```diff
@@ -35,18 +35,29 @@ def filename
 
   def import_rows(filename:)
     file = Tempfile.new(["planning_applications", ".csv"])
-
-    s3.get_object(bucket: "paapi-staging-import", key: filename) do |chunk|
-      file.write(chunk.dup.force_encoding("utf-8"))
-    end
-
+    write_tempfile(file)
     file.close
+
     CSV.foreach(file.path,
                 headers: true, header_converters: :symbol,
                 &method(:import_row))
     file.unlink
   end
 
+  def write_tempfile(file)
+    if ENV["LOCAL_IMPORT_FILE"] == "true"
```

Comment:
I thought it might be better to check for an env var rather than having `if Rails.env.development?` so that both options are available in development.

Reply:
That's even better 🥇